components/home/Banner.tsx

```diff
@@ -9,23 +9,22 @@ import {
 } from '@chakra-ui/react';
 import { useRouter } from 'next/router';
 
-const slate = '#1E293B';
+// const slate = '#1E293B';
 
 function DesktopBanner() {
```

Comment:
You don't need to create separate banners to desktop and mobile. You can just make it adaptive across screens using `grid`